Makefile

```diff
@@ -10,13 +10,29 @@ else
 endif
 
 export VERSION?=$(shell (git describe --abbrev=0 --tags | sed -e 's/v//') || echo $(cat VERSION)-$(git log -1 --pretty='%h'))
-PRERELEASE=`grep -q dev <<< "${VERSION}" && echo "pre" || echo ""`
+PRERELEASE=`echo "${VERSION}" | grep -q dev && echo "pre" || echo ""`
 REVISION?=`git log -1 --pretty='%h'`
 export CRI_DOCKERD_LDFLAGS=-ldflags "-s -w -buildid=${REVISION} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.Version=${VERSION} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.PreRelease=${PRERELEASE} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.GitCommit=${REVISION}"
 
+all: cri-dockerd
+
+# always run go build
+cri-dockerd: FORCE
+	GOARCH=$(ARCH) go build -trimpath $(CRI_DOCKERD_LDFLAGS) -o $@
+FORCE:
+
+prefix ?= /usr/local
+
+install:
+	@mkdir -p $(DESTDIR)$(prefix)/bin
+	install -o root -g root -m 0755 cri-dockerd $(DESTDIR)$(prefix)/bin/cri-dockerd
+	@mkdir -p $(DESTDIR)$(prefix)/lib/systemd/system
+	install packaging/systemd/cri-docker.service packaging/systemd/cri-docker.socket $(DESTDIR)$(prefix)/lib/systemd/system
+	sed -i -e 's,/usr/bin/cri-dockerd,$(prefix)/bin/cri-dockerd,' $(DESTDIR)$(prefix)/lib/systemd/system/cri-docker.service
```

Comment:
I don't think this section is really helpful, as most developers or end users will not want to use it. It /maybe/ makes sense in a container, but at that point you could use a `Dockerfile` instead...

Reply:
Hmm, most software knows how to install? Not sure if running `cri-dockerd` in a container makes sense